programs/svm-spoke/src/instructions/deposit.rs

```diff
@@ -134,45 +134,22 @@ pub fn deposit_v3_now(
     message: Vec<u8>,
 ) -> Result<()> {
     let state = &mut ctx.accounts.state;
```

Comment:
state does not need to be mut here anymore as we only use it for getting current time.

Reply:
`pub fn get_current_time(_state: &State)` expects mutable state for this function. we might want to remove this requirement for this as it does not need to mutate it.